src/configure/templateInputHelper/InputControlProvider.ts

```diff
@@ -148,12 +151,11 @@ export class InputControlProvider {
     }
 
     private _computeMustacheValue(mustacheExpression: string, dependentInputControlArray: InputControl[], dependentClientInputMap: StringMap<any>): string {
-
         var dependentInputValues = this._getInputParameterValueIfAllSet(dependentInputControlArray);
         if (dependentInputControlArray && dependentInputControlArray.length > 0 && !dependentInputValues) {
             return "";
         } else {
-            return MustacheHelper.render(mustacheExpression, { inputs: dependentInputValues, client: dependentClientInputMap });
+            return MustacheHelper.renderObject(mustacheExpression, { inputs: dependentInputValues, client: dependentClientInputMap });
```

Comment:
which case is not working with render()?

Reply:
Just because renderObject is also handling the string scenario and calling render for the same, so updated it just for improvement purpose.